### PR TITLE
[MODULAR] More Ears

### PR DESCRIPTION
**Pull request by Nerev4r**
## About The Pull Request

<details>
<summary> Ears (Around Elves, Watch Yourselves) </summary>

image
image
image
_Wide Elf_ ears from _Sojourn._ I don't have a quip for this.

image
image
image
_Broad Elf_ ears from _Sojourn._ Broad elf, or elf broad?

image
image
image
_Longer Elf_ ears from _Sojourn._ They aren't actually longer on the sides. Gomen nasai.

image
image
image
_Sandfox_ ears from _Virgo._ Yeah.

</details>

## How This Contributes To The Skyrat Roleplay Experience
Adds new customization options borrowed from https://github.com/quotefox/Hyper-Station-13, https://github.com/VOREStation/VOREStation/pulls, https://github.com/Citadel-Station-13/Citadel-Station-13-RP, and https://github.com/sojourn-13/sojourn-station.
## Changelog

:cl:
imageadd: Adds some more ears, five of them.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/ears.dm [hunk 1]
```diff
@@ -321,3 +325,20 @@
...
+	name = "Wide Elf"
+	icon_state = "elfwide"
+
+/datum/sprite_accessory/ears/mutant/elf/broad
+	name = "Broad Elf"
+	icon_state = "elfbroad"
+
+/datum/sprite_accessory/ears/mutant/elf/longer
+	name = "Longer Elf"
+	icon_state = "elflonger"
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/ears.dm]: put these up with the base `/datum/sprite_accessory/ears/mutant/elf` ears

**Review by tf-4 — APPROVED** [review]

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
I like these, the only thing I'd have to say is that the side view of the wide elf ones is very weird, it's as if it represented the ears as being upside down (because it's as if they were wider at the top, when you'd expect them to be wider at the bottom, following the general shape of the ear).

**Comment by GoldenAlpharex** [comment]
Wait, my bad, broad elf.

**Review by GoldenAlpharex — APPROVED** [review]
That's better.

**Pull request merged**